app/models/note.rb

```diff
@@ -6,4 +6,9 @@ class Note < ApplicationRecord
   def logged_at
     created_at.to_date.strftime('%-d %B %Y')
   end
+
+  # @return [Boolean] true if the note is empty
```

Comment:
This is a good example. IMHO the code is very clear, the method name sort of duplicates it. Methods like this can be better if the form a question, even better if you can distill it to a single word. At that point the documentation need only tell us that it is a true/false because the method name should implicitly tell us what is going on.

Reply:
Remove